.eslintrc.js

```diff
@@ -20,6 +20,7 @@ module.exports = {
     "DataView": true,
     "Int8Array": true,
     "Int32Array": true,
-    "Uint8Array": true
+    "Uint8Array": true,
+    "Buffer": true
```

Comment:
We don't use **global.Buffer** anymore, so eslint needs to know about that global

Reply:
Since we're now using `this.Buffer`, we shouldn't need this anymore. Can you remove it?

Reply:
Oups, good call